website/docs/reference/resource-properties/unit-test-input.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Input for unit tests"
+sidebar_label: "Input"
+---
+
+Inputs in unit testing are used to reference a specific model or source for the test:
```

Comment:
```suggestion
You can use inputs in unit testing to narrow the test to a specific model or source using these guidelines:
```

Reply:
I like to avoid "You can" as much as possible in instructional and reference documentation to make directions and guidance sound firm. How about:
```suggestion
Use inputs in your unit tests to reference a specific model or source for the test:
```